src/main/java/com/google/gerrit/extensions/api/projects/Projects.java

```diff
@@ -84,7 +84,6 @@ public enum FilterType {
       ALL
     }
 
-    private final List<String> branches = new ArrayList<>();
```

Comment:
Thanks for your PR!

This file is a plain copy of the official Gerrit source:
https://gerrit.googlesource.com/gerrit/+/master/java/com/google/gerrit/extensions/api/projects/Projects.java

What is the reason for changing this? IMO it would be great if you could implement this functionality based on the existing API.

Reply:
Hello, I'm agree with this philosophy. I've tried to use this list. But in the API documentation https://gerrit-documentation.storage.googleapis.com/Documentation/3.6.2/rest-api-projects.html#project-options
 I can't see the parameter with many branch and how to combine the list to ?b=<branch>.
 
 So I preferred to remove this because is confusant for other developer. I can push other commit to keep this in source code and implement the usage if I know what is used for.

Reply:
You are right, the API really does not seem to support querying multiple branches. I have tried several queries against a real Gerrit instance, and was not able to figure it out.

What I would suggest: revert the change in `Projects`, and throw in our impl in case multiple branches are passed it. If we start to modify things from `com.google.gerrit`, it gets almost impossible to sync the package again in the future because we have to cherry pick changes carefully. I know, this is not the nicest experience when using this API, but I think it should be OK.